src/discof/restore/utils/fd_ssresolve.c

```diff
@@ -272,6 +272,11 @@ fd_ssresolve_parse_redirect( fd_ssresolve_t *        ssresolve,
     }
   }
 
+  if( FD_UNLIKELY( !location ) ) {
```

Comment:
```suggestion
  if( FD_UNLIKELY( !location_len ) ) {
```

Reply:
We typically always do "slice empty?" type checks on the length field. It's a more complete solution because often slices with an invalid (but not NULL) base pointer and a zero size length are considered valid

Reply:
Makes sense!
Should I align https://github.com/firedancer-io/firedancer/blob/8997d39c5f3e0d0b8cf1ac845804ab260fbac4ff/src/discof/restore/utils/fd_sshttp.c#L451-L455 to this pattern as well?

Reply:
yes please